packages/pangea-sdk/PangeaCyber.Net/BaseClient.cs

```diff
@@ -357,7 +365,10 @@ private string PollResultPath(string requestId)
             return $"/request/{requestId}";
         }
 
-        private async Task<HttpResponseMessage> HandleQueued(HttpResponseMessage response)
+        private async Task<HttpResponseMessage> HandleQueued(
+            HttpResponseMessage response,
+            CancellationToken cancellationToken = default
```

Comment:
This is an internal method, is it OK set this default? Shouldn't we check that it's call always with the correct cancellationToken?

Reply:
I think I get what you mean but I don't think it's that important to require that every caller get a cancellation token to give this method. In terms of ensuring tokens are propagated when available, I use [CA2016](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2016).